src/components/page-sidebar/sidebar-tab.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import IconArrow from "../../assets/svg-icons/icon-arrow-circle-left.svg";
+
+import './sidebar-tab.scss';
+
+interface IProps {
+  handleShowSidebarContent: (show: boolean) => void;
+  title: string;
+  sidebarOpen: boolean
+}
+
+export class SidebarTab extends React.PureComponent <IProps>{
+  constructor(props: IProps) {
+    super(props);
+  }
+
+  render() {
+    return(
+      <div className='sideBarTab' onClick={this.handleSidebarClick} data-cy="sidebar-tab">
+        <div className={`icon ${this.props.sidebarOpen ? 'open' : ''}`}>
```

Comment:
Use double quotes inside of here for project consistency.

Reply:
Same with `'sideBarTab'` - use double quotes.  Usually we have the linter detect this in other projects.  We might want to turn that on in the new eslint settings.